packages/backend/src/services/GitIntegrationService/GitIntegrationService.ts

```diff
@@ -164,9 +168,9 @@ export class GitIntegrationService extends BaseService {
         mainBranch,
         token,
         installationId,
-        branchName,
+        branch,
```

Comment:
I renamed some `branch` names for consitency

branchName -> branch : the branch for the new PR
mainBranch : the base branch for the repo (eg: main)